www/src/pages/foundations/spacing.scss

```diff
@@ -0,0 +1,5 @@
+.pgn-doc__spacing-directions .form-check-input {
```

Comment:
Hopefully if we move away from the deprecated `Input`, we can avoid needing to override any CSS related to input styles?

Reply:
yes, done